_includes/unsupported-browsers.html

```diff
@@ -0,0 +1,9 @@
+<div class="no-support">
```

Comment:
Since this file isn‘t included in `_layouts/default.html` (line 6) we can delete it, and bring it back from git history if it’s needed later